modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -107,6 +110,39 @@ export interface CustomGShareGeneratingFunction {
     bitgoToUserCommitment: CommitmentShareRecord;
   }): Promise<GShare>;
 }
+
+export interface CustomMPCv2Round1GeneratingFunction {
```

Comment:
Nit: signing and key gen both have multiple rounds, would be clearer if we specified signing here.
```suggestion
export interface CustomMPCv2SigningRound1GeneratingFunction {
```